.mailmap

```diff
@@ -0,0 +1,20 @@
+Abigail Cabunoc Mayes <abigail.cabunoc@gmail.com>
```

Comment:
What is this file for?

Reply:
`.mailmap` helps to get the contributors that uses different email as the same contributor. So if you filter git history by a name you get all the contributions that x person did with multiple names/emails.
https://git-scm.com/docs/git-check-mailmap